nutanix/resource_nutanix_foundation_ipmi_config.go

```diff
@@ -193,11 +210,9 @@ func resourceFoundationIPMIConfigCreate(ctx context.Context, d *schema.ResourceD
 		nodes := make([]map[string]interface{}, len(v.Nodes))
 		for k1, v1 := range v.Nodes {
 			node := make(map[string]interface{})
-			node["ipmi_configure_successful"] = v1.IpmiConfigureSuccessful
-			node["ipmi_configure_now"] = v1.IpmiConfigureNow
+			node["ipmi_configure_now"] = true
 			node["ipmi_ip"] = v1.IpmiIP
 			node["ipmi_mac"] = v1.IpmiMac
-			node["ipmi_message"] = v1.IpmiMessage
```

Comment:
Why are we removing this ? 
Have we also removed this from schema ?

Reply:
ipmi_message is not a part of model schema so I removed. Seems like I missed removing it from terraform schema. Let me fix it.

Reply:
@siddharth-kulshrestha Lets not remove them, as they are not affecting any flow.